Vizzuality/widget-editor
# update map interactions api

Description of the PR

This pr fixes a problem with our renderer props and how we deal with map interactions. Now you have to specifically disable it with props in the renderer at all times. Default is interactions enabled. 

Also added some basic docs on what properties are available. https://github.com/Vizzuality/widget-editor/wiki/Initializing-the-renderer

In short, now if you don't want interactions on the map you need to specificly check `interactionEnabled={false}`. 

## Testing instructions

Make sure renderer works as expected. You can try setting interactionsEnabled: false in the playground to disable renderer map interactions.

## Pivotal Tracker

none
